community-contributions/joystreamtelegrambot/src/lib/getters.ts

```diff
@@ -0,0 +1,137 @@
+import { Api, Proposals, ProposalArray, ProposalDetail } from "../types";
+import {
+  ChannelId,
+  ElectionStage,
+  PostId,
+  ProposalDetailsOf,
+  ThreadId
+} from "@joystream/types/augment";
+import { Category, CategoryId } from "@joystream/types/forum";
+import { MemberId, Membership } from "@joystream/types/members";
+import { Proposal, ProposalStatus } from "@joystream/types/proposals";
+
+import { formatProposalMessage } from "./announcements";
+import { domain } from "../../config";
+
+// channel
+
+export const currentChannelId = async (api: Api): Promise<number> => {
+  const id: ChannelId = await api.query.contentWorkingGroup.nextChannelId();
+  return id.toNumber() - 1;
+};
+
+export const memberHandle = async (api: Api, id: number): Promise<string> => {
+  const membership: Membership = await api.query.members.membershipById(id);
+  return membership.handle.toJSON();
+};
+
+export const memberHandleByAccount = async (
+  api: Api,
+  account: string
+): Promise<string> => {
+  const id: number = await api.query.members.memberIdsByRootAccountId(account);
```

Comment:
I think we can use `MemberId` here, unless there's a reason not to?

There are also some necessary changes on line 23, 103 and announcements.ts